deployment/templates/fluentd.yaml

```diff
@@ -58,20 +44,15 @@ objects:
             component: "fluentd"
         spec:
           serviceAccountName: aggregated-logging-fluentd
+          nodeSelector:
+            logging-infra: fluentd
           containers:
           - name: fluentd-elasticsearch
-            image: logging-fluentd
+            image: ${IMAGE_PREFIX}logging-fluentd
```

Comment:
In a DC the imagechangetrigger substitutes the real image name. I guess we don't have that in the case of a DaemonSet, it being a kube object. We would need the image suffix too...